### PR TITLE
Adding waffling80, TKL PCB

### PR DESCRIPTION
**Pull request by 4pplet**
## Description

Adding waffling80, TKL PCB. Basically a TKL version of the waffling60. Aiming to give more open source alternatives to the a87/a88 standard

## Types of Changes
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

Thanks! <3

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/4pplet/waffling80/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,
+    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,
+    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,
+    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,
+    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,                            KC_TRNS,    KC_TRNS,    KC_TRNS,
+    KC_TRNS,    KC_TRNS), // extra keys for alps dual action switches
+// extra layer for VIA
```
Comment (fauxpark) [inline keyboards/4pplet/waffling80/keymaps/default/keymap.c:38]:
```suggestion
```

File: keyboards/4pplet/waffling80/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
```
Comment (fauxpark) [inline keyboards/4pplet/waffling80/keymaps/via/keymap.c:16]:
```suggestion
```

File: keyboards/4pplet/waffling80/readme.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# waffling80
+
+A TKL PCB attempting a87 compatibility with different switch and layout-options. 
+
+* Keyboard Maintainer: [4pplet](https://github.com/4pplet)
+* Hardware Supported: [waffling80](https://github.com/4pplet/waffling80)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make 4pplet/waffling60/rev_b:default
```
Comment (fauxpark) [inline keyboards/4pplet/waffling80/readme.md:10]:
```suggestion
    make 4pplet/waffling80:default
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/4pplet/waffling80/readme.md (hunk 3 above)
Comment (4pplet) [reply keyboards/4pplet/waffling80/readme.md]: I updated with "make 4pplet/waffling80/rev_a:default", thanks for the catch! :)

**Comment by 4pplet** [comment]
This should be ready for merge? @fauxpark @zvecr  (no stress😄)

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**